xrpl/wallet/main.py

```diff
@@ -16,69 +17,131 @@ class Wallet:
     details.
     """
 
+    @property
+    def classic_address(self: Wallet) -> str:
+        """
+        The address that publicly identifies this wallet,
+        as a base58 string.
+        """  # noqa: DAR201
+        return self._classic_address
+
+    address = classic_address
+    """
+    The address that publicly identifies this wallet, as a base58 string.
+    """
+
     def __init__(
         self: Wallet,
-        seed: str,
-        sequence: int,
+        public_key: str,
+        private_key: str,
         *,
-        algorithm: Optional[CryptoAlgorithm] = None,
+        master_address: Optional[str] = None,
+        seed: Optional[str] = None,
     ) -> None:
         """
         Generate a new Wallet.
 
         Args:
-            seed: The seed from which the public and private keys are derived.
-            sequence: The next sequence number for the account.
-            algorithm: The algorithm used to encode the keys. Inferred from the seed if
-                not included.
-        """
-        self.seed = seed
-        """
-        The core value that is used to derive all other information about
-        this wallet. MUST be kept secret!
+            public_key: The public key for the account.
+            private_key: The private key used for signing transactions for the account.
+            master_address: Include if a Wallet uses a Regular Key Pair. It must be
+                the master address of the account. The default is `None`.
```

Comment:
"It" is a bit ambiguous (Could be talking about the Wallet/Regular Key Pair/ or master_address)

```suggestion
            master_address: Include if a Wallet uses a Regular Key Pair. This sets the address
                    that this wallet corresponds to. The default is `None`.
```